fastlane/actions/get_latest_github_release.rb

```diff
@@ -0,0 +1,170 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      GET_LATEST_GITHUB_RELEASE_CUSTOM_VALUE = :GET_LATEST_GITHUB_RELEASE_CUSTOM_VALUE
+    end
+
+    class GetLatestGithubReleaseAction < Action
+      def self.run(params)
+        UI.message("Getting latest release on GitHub (#{params[:server_url]}/#{params[:url]})")
+
+        GithubApiAction.run(
+          server_url: params[:server_url],
+          api_token: params[:api_token],
+          api_bearer: params[:api_bearer],
+          http_method: 'GET',
+          path: "repos/#{params[:url]}/releases",
+          error_handlers: {
+            404 => proc do |result|
+              UI.error("Repository #{params[:url]} cannot be found, please double check its name and that you provided a valid API token (if it's a private repository).")
+              return nil
+            end,
+            401 => proc do |result|
+              UI.error("You are not authorized to access #{params[:url]}, please make sure you provided a valid API token.")
+              return nil
+            end,
+            '*' => proc do |result|
+              UI.error("GitHub responded with #{result[:status]}:#{result[:body]}")
+              return nil
+            end
+          }
+        ) do |result|
+          json = result[:json]
+          json.each do |current|
+            Actions.lane_context[SharedValues::GET_GITHUB_RELEASE_INFO] = current
+            UI.message("Version #{current['tag_name']} is latest release live on GitHub.com 🚁")
+            return current
+          end
+        end
+
+        UI.important("Couldn't find GitHub release #{params[:version]}")
+        return nil
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "This will get latest release version that is available on GitHub"
+      end
+
+      def self.details
+        sample = <<-SAMPLE.markdown_sample
```

Comment:
If we end up returning the latest release `name` directly, instead of this blob, we can probably remove this.